github_rules/github_repo_visibility_change.yml

```diff
@@ -1,6 +1,6 @@
 AnalysisType: rule
 Filename: github_repo_visibility_change.py
-RuleID: Github.Repo.VisibilityChange
+RuleID: GitHub.Repo.VisibilityChange
```

Comment:
hmm even though this is a minor change, since you're changing an ID, I can see this potentially creating duplicates, I would caution against changing IDs if we don't need to

Reply:
Agreed

One potential workaround would be to dupe this detection (with the previous RuleID) and change the title to include `--DEPRECATED--` and the YAML to have that detection disabled by default